External/FEXCore/Source/Interface/Core/Dispatcher/RISCVDispatcher.cpp

```diff
@@ -0,0 +1,54 @@
+#include "Interface/Core/LookupCache.h"
+
+#include "Interface/Core/ArchHelpers/MContext.h"
+#include "Interface/Core/Dispatcher/RISCVDispatcher.h"
+#include "Interface/Core/Interpreter/InterpreterClass.h"
+
+#include "Interface/Context/Context.h"
+
+#include <FEXCore/Core/CPUBackend.h>
+#include <FEXCore/Core/CoreState.h>
+#include <FEXCore/Core/X86Enums.h>
+#include <FEXCore/Debug/InternalThreadState.h>
+#include <FEXHeaderUtils/Syscalls.h>
+
+#include <array>
+#include <bit>
+#include <cmath>
+#include <cstdint>
+#include <memory>
+#include <stddef.h>
+
+#include <biscuit/assembler.hpp>
+
+#include <sys/mman.h>
+#include <sys/syscall.h>
+#include <unistd.h>
+
+namespace FEXCore::CPU {
+#ifdef _M_RISCV_64
+static constexpr size_t MAX_DISPATCHER_CODE_SIZE = 4096;
+
+RISCVDispatcher::RISCVDispatcher(FEXCore::Context::Context *ctx, FEXCore::Core::InternalThreadState *Thread, uint8_t* Buffer)
+  : Dispatcher(ctx, Thread), RISCVEmitter(ctx, Buffer, MAX_DISPATCHER_CODE_SIZE) {
+
+
+  biscuit::Label LoopTop;
+  Bind(&LoopTop);
+  J(&LoopTop);
+
+  DispatchPtr = (CPUBackend::AsmDispatch)((uintptr_t)Buffer + LoopTop.GetLocation().value());
```

Comment:
@lioncash Might want to look at letting Biscuit get an absolute cursor pointer location from the Assembler.